dmci/api/worker.py

```diff
@@ -104,48 +100,60 @@ def validate(self, data):
             if not valid:
                 return valid, msg, data
 
+            if (".dev" in self._namespace and self._conf.env_string != "dev") or (
```

Comment:
why not just check
```suggestion
            if (self._conf.env_string not in self._namespace) or (
```
and so on? Does it fail for prod then?

Reply:
yes

Reply:
I see that this will also be wrong. The point is that I prefer to avoid hardcoding "dev" and "staging"...